.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+<!--Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.-->
+
+## Merge requirements
+
+Please take into account the following non-code changes that you may need to make with your PR:
+
+- [ ] CHANGELOG entry for user-facing changes
```

Comment:
I didn't even know we had a CHANGELOG, and, judging by its contents, few people did 😅

Reply:
Yup, exactly 😂